content/en/docs/home/_index.md

```diff
@@ -56,8 +56,8 @@ cards:
   description: Anyone can contribute, whether you're new to the project or you've been around a long time.
   button: Contribute to the docs
   button_path: /docs/contribute
-- name: release-notes
-  title: K8s Release Notes
+- name: Download
+  title: Downloads
   description: If you are installing Kubernetes or upgrading to the newest version, refer to the current release notes.
```

Comment:
We should change this text too.

Reply:
Sounds good! Thanks @sftim !

Reply:
Once you have edited:
> If you are installing Kubernetes or upgrading to the newest version, refer to the current release notes.

then we can progress this PR.

Reply:
@sftim It's done now!